rdrf/rdrf/settings.py

```diff
@@ -646,3 +646,4 @@
 # URLs whitelisted for meeting the security conventions
 # Refer to docs/security/README.rst
 SECURITY_WHITELISTED_URLS = url_whitelist.SECURITY_WHITELISTED_URLS
+MAX_SESSION_REFRESH_RETRIES = 5
```

Comment:
Should be configurable with an env variable as well.

Reply:
Updated.